Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -10,11 +10,15 @@
     <DebugType>Full</DebugType>
     <NoWarn>CS8002</NoWarn>
   </PropertyGroup>
-  
+
   <PropertyGroup Condition="'$(TargetFramework)' == 'uap10.0.16299'">
     <DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);WINRT</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">
+    <DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);WINFORMS</DefineConstants>
```

Comment:
I don't think you need to define a new constant, `NET462` should be defined and can be used instead